features/providers/civil_application_journey.feature

```diff
@@ -16,108 +16,192 @@ Feature: Civil application journeys
     Then I click find address
     Then I select an address '3, LONSDALE ROAD, BEXLEYHEATH, DA7 4NG'
-    Then I click continue
+    Then I click "Continue"
     Then I am on the benefit check results page
     Then I see a notice saying that the citizen receives benefits
 
   @javascript @vcr
   Scenario: I can see that the applicant does not receive benefits
-    Given I visit the application service
-    And I click link "Start"
-    And I click link "Start now"
-    And I search for proceeding 'Application for a care order'
-    Then proceeding suggestions has results
-    Then I select and continue
+    Given I start the journey as far as the applicant page
     Then I enter name 'Test', 'Paul'
     Then I enter the date of birth '10-12-1961'
     Then I enter national insurance number 'JA293483B'
-    Then I click continue
+    Then I click "Continue"
     Then I am on the postcode entry page
-    Then I enter a valid postcode 'DA74NG'
+    Then I enter a postcode 'DA74NG'
     Then I click find address
     Then I select an address '3, LONSDALE ROAD, BEXLEYHEATH, DA7 4NG'
-    Then I click continue
+    Then I click "Continue"
     Then I am on the benefit check results page
     Then I see a notice saying that the citizen does not receive benefits
 
+  @javascript @vcr
+  Scenario: I want to change first name from the check your answers page
+    Given I complete the journey as far as check your answers
+    And I click Check Your Answers Change link for 'First name'
+    Then I enter the first name 'Bartholomew'
+    Then I click "Continue"
+    Then I should be on the Check Your Answers page
+    And the answer for 'First name' should be 'Bartholomew'
+
+  @javascript @vcr
+  Scenario: I want to return to the check your answers page without changing first name
+    Given I complete the journey as far as check your answers
+    And I click Check Your Answers Change link for 'First name'
+    Then I click link "Back"
+    Then I should be on the Check Your Answers page
+
+  @javascript @vcr
+  Scenario: I want to change the proceeding type from the check your answers page
+    Given I complete the journey as far as check your answers
+    And I click Check Your Answers Change link for 'Proceeding Type'
+    And I search for proceeding 'Application for a care order'
+    Then proceeding suggestions has results
+    Then I select and continue
+    Then I should be on the Check Your Answers page
+
+  @javascript @vcr
+  Scenario: I want to return to the check your answers page without changing proceeding type
+    Given I complete the journey as far as check your answers
+    And I click Check Your Answers Change link for 'Proceeding Type'
+    And I search for proceeding 'Application for a care order'
+    Then proceeding suggestions has results
+    Then I click link "Back"
+    Then I should be on the Check Your Answers page
+
+  @javascript @vcr
+  Scenario: I want to change email from the check your answers page
```

Comment:
With `email` now being on **Client's details** page, maybe we don't need this scenario and the one above. They are similar to the **first name** one